package.json

```diff
@@ -65,6 +65,7 @@
     "ts-node": "9.1.1",
     "typescript": "4.1.3",
     "web-vitals": "1.1.1",
+    "yarn": "1.22.10",
```

Comment:
`yarn` should preferably be a global dependency and not a local one. This line indicates that you have installed `yarn` without setting the global (`-g`) flag.